examples/backupRTC/backupRTC.ino

```diff
@@ -0,0 +1,104 @@
+/*
+  F1DateRetention
```

Comment:
```suggestion
  F1RTCDateRetention
```

And please rename the directory and ino file. as well.

Reply:
changed and renamed